### DISCUSSION TITLE
Dashboard feed should be configurable to show activity only from repositories you work in or opt-in to

### DISCUSSION THREAD

**Discussion by charukiewicz** [community/community#111010] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I've been using Github for over 10 years on a single account, and I've never found the the dashboard feed materially useful or a feature that gives me information I want to see. The recent update hasn't changed this. I've played with the settings and still can't get it to do what I want, which is show things that are actually relevant to me on the basis of what I am contributing to (plus what I want to see).

When I and I'm sure many others go to github.com, the useful content on dashboard is mostly just the 300px wide sidebar, with the feed and the right hand sidebar serving almost no utility for how I want to use Github. It seems that up until recently, Github would show me activity from the hundreds of repositories I had starred, as well as people I followed, which is a bunch of noise I really don't care about.

But this doesn't have to be this way. The feed could be an incredibly useful feature, if it could be made to consist of things that are actually relevant to you.

Here's what I want to be able to set the feed to contain (ideally this should all be configurable):

- Commits/new issues/new pull requests/new comments to repositories I own or am a contributor of
- Commits/new issues/new pull requests/new comments to repositories that are in organizations I'm apart of
- Commits/new issues/new pull requests/new comments to repositories I want to see activity from
- A special highlight for anything that I am @-tagged in.
- A special highlight for replies to issues I have created.
- A special highlight for replies to pull requests I have created.

In my ideal setup, I want to turn off everything that the feed currently shows, and opt-in to only the above list of features. I think this would make the dashboard incredibly valuable and a productivity booster.

The dashboard feed could be such a useful feature. Please consider adding the ability to customize it with the options above.

------------------------

There's a similar discussion about this in #110596, but that thread is framed very critically.

**Comment by phansch** [comment]
Yeah, I was having a similar feeling. When I go to github.com, I don't want to discover repos, I want to work on the things that need to be worked on. Ideally without having to change filter settings hidden somewhere in the UI.

**Comment by kothurain** [comment]
All before facebook and Facebook page safe setting return request add please
